packages/system/gpu-operator/charts/gpu-operator/charts/node-feature-discovery/templates/prometheus.yaml

```
@@ -0,0 +1,26 @@
+    {{ toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  podMetricsEndpoints:
+    - honorLabels: true
+      interval: {{ .Values.prometheus.scrapeInterval }}
+      path: /metrics
+      port: metrics
+      scheme: http
+  namespaceSelector:
+    matchNames:
+    - {{ include "node-feature-discovery.namespace" . }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)